.travis.yml

```diff
@@ -13,7 +13,6 @@ before_install:
 install:
   - npm ci
   - bash ./scripts/build-dependencies.sh
-  - git clone --branch v`cat package.json | python -c "import sys, json; print(json.load(sys.stdin)['googleLibphonenumberVersion'])"` https://github.com/googlei18n/libphonenumber/
```

Comment:
i would not suggest not removing this

Reply:
As I moved this line to `./scripts/build-dependencies.sh`. Do we need to add it again in travis.yml?

Reply:
nah, its okay